app/uk/gov/hmrc/eoricommoncomponent/frontend/views/helpers/inputRadioGroup.scala.html

```diff
@@ -23,10 +23,12 @@
         legendAsHeading: Boolean = true,
         hint: Option[Hint] = None,
         classes: String = "govuk-fieldset__legend--xl",
-        inline: Boolean = false
+        inline: Boolean = false,
+        formGroupClasses: String = ""
```

Comment:
Added this to be in line with https://github.com/hmrc/address-lookup-frontend. By default it's empty `String`